.travis.yml

```diff
@@ -1,16 +0,0 @@
-sudo: required 
-dist: trusty
-language: go
-go:
-  - "1.11"
-
-install:
-  - go get -t -v -d ./...
-
-script:
-  # Check whether files are syntactically correct.
-  - "gofmt -l $(find . -name '*.go' | tr '\\n' ' ') >/dev/null"
-  # Check whether files were not gofmt'ed.
-  - "gosrc=$(find . -name '*.go' | tr '\\n' ' '); [ $(gofmt -l $gosrc 2>&- | wc -l) -eq 0 ] || (echo 'gofmt was not run on these files:'; gofmt -l $gosrc 2>&-; false)"
-  - go tool vet .
-  - go test -c -race . && sudo ./renameio.test -test.v
```

Comment:
I don’t think we have parity with this line yet:

https://github.com/google/renameio/runs/1080834683?check_suite_focus=true

* we don’t use -race in GitHub actions
* we don’t use -test.v in GitHub actions
* we don’t use sudo (required for some tests AFAIR) in GitHub actions.

Reply:
I *think* that `-race` doesn't have any practical effect for testing this package as none of the tests create any goroutines, but I'm happy to be wrong.

The tests also pass without `sudo` on GitHub Actions.

For the `gofmt` test, I'll open a PR that adds a `golangci-lint` test which includes a format check.

Reply:
> The tests also pass without `sudo` on GitHub Actions.

Yeah, because they’re skipped:
```
		t.Skipf("cannot mount tmpfs on %s: %v", mount1, err)
```

Can you add `sudo` and `-v` please, so that we see which tests are run?

Reply:
Done in #25 (this PR here is based on an older `master` which does not include the GitHub Action).